Packages/Tracking/Examples~/XR Examples/Example Assets/Physical Hands/Prefabs/Physical Hands Button 2.0.prefab

```diff
@@ -0,0 +1,513 @@
+%YAML 1.1
```

Comment:
This prefab should:

1. be in the core plugin
2. not need 2.0 in the name (we could rename the old one?)
3. be added to the prefab create menu (will automatically happen by doing steps 1 & 2)